products/podman.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
```

Comment:
```suggestion
title: Podman
category: app
iconSlug: podman
permalink: /podman
```

As advised in https://github.com/containers/podman/discussions/23352#discussioncomment-10112901 and because this name is not used on https://podman.io/ (already renamed the file).